pkgs/development/python-modules/databricks-connect/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, six, py4j }:
+
+buildPythonPackage rec {
+  pname = "databricks-connect";
+  version = "7.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "996a9d0f271f6c7edbd2d85b2efb6ff4e58d15222e80f87ca17fdbf224e17056";
+  };
+
+  sourceRoot = ".";
+
+  propagatedBuildInputs = [ py4j six pkgs.jdk ];
+
+  # User have to run tests command 
+  #   databricks-connect test
+  # manually after install and configuration
+  #   databricks-connect configure
```

probably more succinct
```suggestion
  # requires network access
```